crate/azure_sev_quote/src/main.rs

```diff
@@ -0,0 +1,14 @@
+use std::error::Error;
+
+use azure_sev_quote::get_quote_from_tpm;
+
+pub fn main() -> Result<(), Box<dyn Error>> {
+    let quote = get_quote_from_tpm();
+    println!("quote: {quote:?}");
+
+    if let Ok(quote) = quote {
+        std::fs::write("azure_sev_quote.bin", quote).expect("can't write quote to file!");
```

Comment:
if you `expect` here, there is no need to return a `Result` in this `main` function

Reply:
Removed this file which was used for testing on Azure VM.